lib/lte_link_control/Kconfig

```diff
@@ -54,28 +54,35 @@ config LTE_UNLOCK_PLMN
 	help
 		Disable PLMN locks for network selection.
 
+menuconfig LTE_PSM_REQ
+	bool "Enable requesting use of PSM."
+	help
```

Comment:
I think this option should be `default y` to avoid breaking existing applications.
In addition, you can introduce a `CONFIG_LTE_PSM_REQ_ON_INIT` option that is used to ifdef in `lte_lc_psm_enable()` whn it's enabled. That way we existing applications should not be affected as much, without changing their behavior.

Reply:
Thanks for your idea :) For eDRX you are absolutely right. For PSM I disagree. If you look at the current code, only eDRX is initialized within init_and_config function. PSM is not active by default and therefore I also would disable PSM by default.